doc/README.md

```diff
@@ -237,10 +237,45 @@ them. This is useful to combine profiles from multiple processes of a
 distributed job. The profiles may be from different programs but must be
 compatible (for example, CPU profiles cannot be combined with heap profiles).
 
-pprof can subtract a profile from another in order to compare them. For that,
-use the **-base= _profile_** option, where *profile* is the filename or URL for the
-profile to be subtracted. This may result on some report entries having negative
-values.
+
+## Comparing profiles
```

Comment:
It is a bit strange that "Comparing profiles" is a section under "Fetching profiles", is it intentional? I'd expect it to be more related to viewing than to fetching.

Reply:
Makes sense. Moved.